src/server/graphql/models/Team/archiveTeam/archiveTeam.js

```diff
@@ -0,0 +1,58 @@
+import getRethink from 'server/database/rethinkDriver';
+import {
+  requireSUOrTeamMember,
+  requireWebsocket
+} from 'server/utils/authorization';
+import {handleSchemaErrors} from 'server/utils/utils';
+import {
+  GraphQLNonNull,
+  GraphQLBoolean,
+} from 'graphql';
+import {TeamInput} from '../teamSchema';
+import shortid from 'shortid';
+import {TEAM_ARCHIVED} from 'universal/utils/constants';
+import archiveTeamValidation from './archiveTeamValidation';
+
+export default {
+  type: GraphQLBoolean,
+  args: {
+    updatedTeam: {
+      type: new GraphQLNonNull(TeamInput),
+      description: 'The input object containing the teamId and any modified fields'
+    }
+  },
+  async resolve(source, {updatedTeam}, {authToken, socket}) {
+    const r = getRethink();
+
+    // AUTH
+    requireSUOrTeamMember(authToken, updatedTeam.id);
```

Comment:
I think we want to this to be `requireSUOrLead`, we wouldn't just anybody to be able to archive a team, right?